lib/resources/pg_ident_conf.rb

```diff
@@ -0,0 +1,88 @@
+# encoding: utf-8
+# copyright: 2017
+# author: Rony Xavier,  rx294@nyu.edu
+# author: Aaron Lippold, lippold@gmail.com
+# license: All rights reserved
+
+require 'resources/postgres'
+
+module Inspec::Resources
+  class PGIdentConf < Inspec.resource(1)
+    name 'pg_ident_conf'
+    desc 'Use the pg_ident_conf InSpec audit resource to test the client
+          authentication data is controlled by a pg_ident.conf file.'
+    example "
+      describe pg_ident_conf.where { pg_username == 'acme_user' } do
+        its('map_name') { should eq ['ssl-test'] }
+      end
+    "
+
+    attr_reader :params, :conf_dir, :conf_file
+
+    def initialize(ident_conf_path = nil)
+      @conf_dir = inspec.postgres.conf_dir
+      @conf_file = ident_conf_path || File.expand_path('pg_ident.conf', inspec.postgres.conf_dir)
+      @files_contents = {}
```

Comment:
`@files_contents` is set to an empty hash here, but later when the variable is populated, it's populated with an array. This should likely either be set to `nil` or an empty array rather than a hash. Ultimately, since this variable isn't really used anywhere, I'd eliminate it completely.